plugins/flutter_aepassurance/pubspec.yaml

```diff
@@ -13,7 +13,11 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  flutter_aepcore: ">= 1.0.0 <2.0.0"
+  flutter_aepcore: ">= 2.0.0 <3.0.0"
+
+dependency_overrides:
```

Comment:
Let us release `flutter_aepcore` first and then remove `dependency_overrides` section from each of the individual packages.